org.jasp-stats.JASP.json

```diff
@@ -0,0 +1,5548 @@
+{
+	"app-id": "org.jasp-stats.JASP",
+	"runtime": "org.kde.Sdk",
+	"runtime-version": "5.12",
+	"sdk": "org.kde.Sdk",
+	"command": "org.jasp-stats.JASP",
+	"finish-args":[
+		"--socket=x11",
+		"--socket=wayland",
+		"--share=ipc",
+		"--share=network",
+		"--filesystem=home",
+		"--env=PATH=/usr/bin:/app/bin"
+	],
+	"build-options": {
```

Comment:
No need for those build-options. The cflags & cxxflags are already set by default in freedesktop sdk 18.08.